custom/custom.go

```diff
@@ -0,0 +1,17 @@
+package custom
+
+import "github.com/ONSdigital/dp-api-clients-go/v2/dataset"
+
+func GenerateCustomTitle(dims []dataset.VersionDimension) string {
+	var title string
+	l := len(dims)
+	for i, d := range dims {
+		if i == (l - 1) {
```

Comment:
So maybe this should be as follows:

```suggestion
		if i > 0 && i == lastIndex {
```